### DISCUSSION TITLE
Homkit Code

### DISCUSSION THREAD

**Discussion by Gandalf8266** [Sunoo/homebridge-camera-ffmpeg#1417] [General]
Hello, which HomeKit Code must be used, to add a camera in my home app.
I the past, I used the Code 031-45-154 but this don't work any more, I seems the syntax has changed, because it doesn't fit in the syntax like in the past.
Maybe after the HomeKit Upgrade in the apple home app, there has changed something.
![IMG_7742](https://github.com/Sunoo/homebridge-camera-ffmpeg/assets/38372908/47c876e0-78cf-4a57-b6ea-63698ea3f2a1)
![IMG_7743](https://github.com/Sunoo/homebridge-camera-ffmpeg/assets/38372908/99c6a3ae-66ab-44ca-8c14-c2554e05db9c)
![IMG_7744](https://github.com/Sunoo/homebridge-camera-ffmpeg/assets/38372908/8b1a2ea6-57a4-4788-97cd-3efcd4bb8c88)

**Comment by Sunoo** [comment]
It should be the same as what you used for Homebridge.

**Reply by Gandalf8266** [reply]
thx, that works.

**Reply by Gandalf8266** [reply]
Hello, I moved the homebridge from a raspberry to a Docker container on my synology. Now I have the same Problem, I can't neither with the code above nor with the homebridge add the camera.
Any Idea?

**Reply by Gandalf8266** [reply]
After waiting two days, the code works, really strange.
